app/README.md

```diff
@@ -0,0 +1,5 @@
+# Mautic Core-lib repo
+
+## This is a mirror repository of the /app folder of Mautic, and is managed centrally in https://github.com/mautic/mautic/blob/head/app.  This is a read-only mirror repository.
+
+**📣 Please make PRs and issues against Mautic Core, not here!**
```

Comment:
I'm not sure everyone will get what is meant by Mautic Core as this will be in the repository called core-lib so it may be confusing.
```suggestion
**📣 Please make PRs and issues against the [mautic/mautic](https://github.com/mautic/mautic) repository, not here!**
```